nltk/stem/lancaster.py

```diff
@@ -304,7 +319,14 @@ def __applyRule(self, word, remove_total, append_string):
             word += append_string
         return word
 
+    def strip_prefix(self, word):
+        """Remove prefix from a word.
+        """
```

Comment:
Better yet, let's also try to track down the history of how it got added to Whoosh via their source control. For all we know it may have been a copy-and-paste to begin with from some other source that we're failing to credit! Knowing the history will also let us credit individual authors in CONTRIBUTORS.md.

None of this is vital, but would be nice (and also potentially *practically* useful in helping understand the alterations that were made to the original algorithm and what motivated them).